apps/dashboard/src/core-ui/batch-upload/batch-lazy-mint.tsx

```diff
@@ -269,13 +267,12 @@ export const BatchLazyMint: ComponentWithChildren<BatchLazyMintProps> = (
                 mb={2}
               >
                 <div className="flex flex-row">
-                  <Icon
-                    boxSize={5}
-                    as={IoChevronBack}
-                    color="gray.600"
+                  <Button
+                    className="text-muted-foreground"
                     onClick={() => setStep(0)}
-                    cursor="pointer"
-                  />
+                  >
+                    <ChevronLeftIcon className="size-5 cursor-pointer" />
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/0524ed44-1f67-4c5a-999e-5785cb25f977.png)

Need to fix the style and alignment of back button here. Button should be "ghost" variant and vertically aligned

Reply:
also- use the new shadcn button instead of chakra here

Reply:
oops, let me fix the style. I will eventually replace the chakra button in another PR

Reply:
Didn't check the style after updating the code - mb here